components/TypeForm/PopUp.tsx

```diff
@@ -12,18 +12,15 @@ export type PopUpProps = PopupOptions & {
 const PopUp = ({
   id, children, className, ...rest
 }:PopUpProps) => {
-  const ref = useRef<HTMLButtonElement>(null);
 
   const handleClose = () => {
-    if (ref.current) {
-      ref.current.focus();
-    }
+    document.getElementById(id)?.focus();
   };
 
   return (
     <PopupButton
       onClose={handleClose}
-      buttonProps={{ type: 'button', ref: ref, className: className }}
```

Comment:
Button props no longer supported the ref prop so I just moved it to a getElementById...

Reply:
Huh I wonder why they removed that support